### DISCUSSION TITLE
Local Deployment

### DISCUSSION THREAD

**Discussion by jeffreyhardinggadoe** [microsoft/sample-app-aoai-chatGPT#1031] [Q&A]
Hi all. I downloaded all of the files to my local machine and edited the .env file as instructed. I included all of the information for our company's endpoints etc... When I run start start.cmd from VS, everything appears to run bug free, but I don't see a new webapp anywhere in my Azure account. Am I looking in the wrong place? I tried to view it using this link from the documentation, but it doesn't show anything.

http://127.0.0.1:50505/.

**Comment by RezaMatin** [comment]
Try using docker local:
docker build . --file WebApp.Dockerfile -t imagename

**Comment by gaborvar** [comment]
If you plan to deploy to Azure with the blue "Deploy to Azure" button then it is a Docker based deployment (see deployment.json in infrastructure directory). UI customization is not included and there are other issues.  
If you use the "az webapp" commands as detailed under "Deploy with the Azure CLI" then it is more complete deployment. Do not forget to run the Powershell script in pwsh (not in Windows Powershell) which creates the json for the environment variables. (bash script is also provided)  Then point your browser to the address that you used in the deployment process.
